apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-template-example.component.ts

```diff
@@ -0,0 +1,29 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ValidatorFn, Validators } from '@angular/forms';
+import { FdDate, FdDatetime } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-datetime-picker-template-example',
+    templateUrl: './platform-datetime-picker-template-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDatetimePickerTemplateExampleComponent {
+    storedDate: FdDatetime = FdDatetime.getToday();
+
+    requiredDateValidator: ValidatorFn[] = [Validators.required];
```

Comment:
Sorry for the late comment.
But are you sure that you need this parameter? I wasn't able to find it's usage

Reply:
have removed this param, please check, thanks!